omniscidb/ArrowStorage/ArrowStorage.cpp

```diff
@@ -715,7 +715,8 @@ void ArrowStorage::appendArrowTable(std::shared_ptr<arrow::Table> at, int table_
         switch (col_arr->type()->id()) {
           case arrow::Type::STRING:
             // if the dictionary has already been materialized, append indices
-            if (!config_->storage.enable_lazy_dict_materialization ||
+            if (config_->storage.enable_non_lazy_data_import ||
```

Comment:
Is this necessary simply to force non-lazy data import in all cases when non lazy data import is enabled in config (vs just lazy dict materialization), or is there a codepath here that is important if we want non-lazy data import and lazy dict materialization is already disabled?

Reply:
I did this change assuming that lazy dictionaries are going to be enabled by default soon and I would like to disable all lazy import features with a single flag in benchmarks.